pkg/agent/utils.go

```diff
@@ -379,13 +379,13 @@ func IsKubeletConfigFileEnabled(cs *datamodel.ContainerService, profile *datamod
 }
 
 // IsKubeletClientTLSBootstrappingEnabled get if kubelet client TLS bootstrapping is enabled
-func IsKubeletClientTLSBootstrappingEnabled(cs *datamodel.ContainerService, profile *datamodel.AgentPoolProfile, kubeletClientTLSBootstrappingEnabled bool) bool {
+func IsKubeletClientTLSBootstrappingEnabled(tlsBootstrapToken *string, kubeletClientTLSBootstrappingEnabled bool) bool {
 	if !kubeletClientTLSBootstrappingEnabled {
```

Comment:
Do we need this? can we just do return tlsBootstrapToken != nil ? 

Once a cse is baked into VM, there is no way to flip the toggle right?

Reply:
synced offline, removed the redundant feature flag.